src/build/arg/mod.rs

```diff
@@ -2071,6 +2071,46 @@ impl<'help> Arg<'help> {
         self.takes_value(true).multiple_values(true)
     }
 
+    /// Specifies that option values that are invalid UTF-8 should *not* be treated as an error.
+    ///
+    /// **NOTE:** Using argument values with invalid UTF-8 code points requires using
+    /// [`ArgMatches::value_of_os`], [`ArgMatches::values_of_os`], [`ArgMatches::value_of_lossy`],
+    /// or [`ArgMatches::values_of_lossy`] for those particular arguments which may contain invalid
+    /// UTF-8 values.
+    ///
+    /// **NOTE:** Setting this requires [`ArgSettings::TakesValue`]
+    ///
+    /// # Examples
+    ///
+    #[cfg_attr(not(unix), doc = " ```ignore")]
+    #[cfg_attr(unix, doc = " ```rust")]
```

Comment:
```suggestion
    #[cfg_attr(unix, doc = " ```")]
```

Reply:
There a reason why you prefer the default language choice enough to call it out?

This is also being consistent with all of the code around it

Reply:
Personally, I'm the opposite and prefer being explicit about it :)

One nice thing about being explicit is you don't have to think about what context you are in and if it matters (e.g. README vs .rs)

Reply:
In that case, shouldn't we have `rust,ignore` instead of `ignore` in the above line?

In the discussion of having `rust` or not, I am not exactly pedantic. I was trying to figure what's the better way and context definitely matters.

Reply:
I'm speaking in terms of my ideals and not what was done to be consistent / what was copy pasted.